packages/spor-react/src/theme/slot-recipes/tabs.ts

```diff
@@ -31,6 +31,35 @@ export const tabsSlotRecipe = defineSlotRecipe({
     },
   },
   variants: {
+    fitted: {
+      true: {
+        list: {
+          display: "flex",
```

Comment:
Denne må ha en verdi for at den skal fungere.

```suggestion
          display: "flex",
          width: "fit-content",
```